### ISSUE TITLE
Do not recreate triple slash comments if already inside one

### ISSUE THREAD

**Issue by fingeg** [fingeg/DoxygenComments#10]
Because in a triple slash comment every line starts like the first line, the plugin thinks it had to create a new comment every time

**Comment by echubarov** [comment]
@fingeg Has this not been attended to yet?

**Comment by fingeg** [comment]
No, not yet. I am very busy right now, but pull requests are always welcome
